tests/federation_room_join_partial_state_test.go

```diff
@@ -2405,6 +2405,77 @@ func TestPartialStateJoin(t *testing.T) {
 			mustSyncUntilDeviceListsHas(t, alice, syncToken, "left", server.UserID("elsie"))
 			mustQueryKeysWithFederationRequest(t, alice, userDevicesChannel, server.UserID("elsie"))
 		})
+
+		// test that device lists stop being tracked when leaving a partial state room before the
+		// partial state join completes.
+		t.Run("Device list no longer tracked when leaving partial state room", func(t *testing.T) {
+			t.Skip("Cannot yet leave a room during resync")
```

Comment:
to be clear: if these are both skipped, does that mean you've not been able to try them?

It might be good to link to the bug that they are waiting for (https://github.com/matrix-org/synapse/issues/12802, I guess?)